Apps/W1/DynamicsGPHistoricalData/app/src/Pages/HistInventoryTrxHeaders.Page.al

```diff
@@ -22,47 +22,38 @@ page 41012 "Hist. Inventory Trx. Headers"
             {
                 field("Document No."; Rec."Document No.")
                 {
-                    ApplicationArea = All;
```

Comment:
We should not remove property from the fields. It may be redundant, but test well on the actual deployment. It may hide the fields.

Reply:
Yes redundant, but I've added them back to be safe.